### DISCUSSION TITLE
Sponsorship: can you do it less?

### DISCUSSION THREAD

**Discussion by ildar-dev** [stoqey/ib#128] [General]
I am just a student from Russia, but I want to support your projects. Is it possible. to donate an amount, but not the cost in my life?

Thank you in advance.

<img width="356" alt="image" src="https://user-images.githubusercontent.com/42977192/140504731-b7694460-1b29-4594-84b0-582b121281a2.png">

**Comment by mfrener** [comment]
@ceddybi will be able to answer ;)

But think it is more aimed toward professionals that make money using that code (share some with authors pls). As a student, hobbyist, semi-pro just go ahead and use it.. ;) and remember about pushing sponsor button when you have hit the million profit on your trading account :D :P

**Comment by ceddybi** [comment]
@IldarDavletyarov All is done, your lordship! I just want to thank you very much, Sir, you're far too kind ❤️ ❤️ ❤️
